homeassistant/components/shelly/__init__.py

```diff
@@ -145,12 +145,12 @@ def __init__(self, wrapper: ShellyDeviceWrapper, block):
         self._name = f"{self.wrapper.name} - {self.block.description.replace('_', ' ')}"
 
     @property
-    def name(self):
+    def name(self) -> str:
```

Comment:
```suggestion
    def name(self):
```

Reply:
We don't want to specify types? I thought this will be helpful.